Client/Frontend/Browser/Playlist/Cells/PlaylistCellRedacted.swift

```diff
@@ -0,0 +1,230 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Brandon on 2022-07-21.
+//
+
+import SwiftUI
+import CoreData
+import Shared
+import BraveUI
+import DesignSystem
+import BraveShared
+import Data
+
+@available(iOS, deprecated: 15.0, renamed: "SwiftUI.AsyncImage")
+public struct AsyncImage<Content: View>: View {
+  @StateObject private var loader: ImageLoader
+  
+  private let url: URL?
+  private let scale: CGFloat
+  private let transaction: Transaction
+  private let render: (AsyncImagePhase) -> Content
+  
+  public init(url: URL?, scale: CGFloat = 1) where Content == Image {
+    self.url = url
+    self.scale = scale
+    self.transaction = Transaction()
+    _loader = StateObject(wrappedValue: ImageLoader())
```

Comment:
`loader` can be assigned at it's declaration. Though it typically works, Apple recommends not using this initializer directly: https://developer.apple.com/documentation/swiftui/stateobject/init(wrappedvalue:)

Reply:
Fixed: https://github.com/brave/brave-ios/pull/5798/commits/db40c42a11edc78866fbec8178ed48483df5761e